Source/Initialization/WarpXInitData.cpp

```diff
@@ -105,6 +105,145 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
     }
 }
 
+void
+WarpX::PrintMainPICparameters ()
+{
+    amrex::Print()<<"-------------------------------------------------------------------------------"<<"\n"; //oshapoval
+    amrex::Print()<<"------------------------- MAIN EM PIC PARAMETERS ------------------------------"<<"\n"; //oshapoval
+    amrex::Print()<<"-------------- (for more details refer to output.txt file) --------------------"<<"\n"; //oshapoval
```

Comment:
The `output.txt` file is where this output will end in.

I think you refer to a future file that we discussed we want to create that always contains all inputs options (PR not yet open for this one).

Reply:
Correct, thanks for noticing this. Will be removed within this PR. It was just to have better overview on how command line output can look like within this and future changes.

Reply:
Fixed.